fsharp-samples/getting-started/load.fsx

```diff
@@ -0,0 +1,16 @@
+[<AutoOpen>]
+module Load
+
+// First reference net standard and then load in all the assemblies used by ML .NET.
+#r "netstandard"
+#load @"..\.paket\load\netstandard2.0\main.group.fsx"
+
+// This code below is required in order to make native assemblies used by ML .NET visible to F#'s interactive prompt for scripts.
+// It's not required if you create a console / web application etc.
+let private nativeDirectory = System.IO.Path.Combine(System.Environment.GetEnvironmentVariable "userprofile", @".nuget\packages\microsoft.ml\0.2.0\runtimes\win-x64\native")
+System.Environment.SetEnvironmentVariable("Path", System.Environment.GetEnvironmentVariable "Path" + ";" + nativeDirectory)
+
+module Datasets =
+    let private buildPath file = System.IO.Path.Combine(@"..\datasets\", file)
```

Comment:
I'm just wondering why this path is pointing to just one folder up instead of two folders up like `..\..\datasets\`. Currently I'm struggling a bit because compiler is looking for `dataset` in `getting-started` directory. But maybe I'm doing something wrong.